tests/drivers/flash_simulator_stub/prj.conf

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+CONFIG_ZTEST=y
+CONFIG_FLASH=y
+CONFIG_FLASH_SIMULATOR=n
+CONFIG_FLASH_SIMULATOR_STUB=y
+CONFIG_FLASH_SIMULATOR_STUB_UNALIGNED_READ=n
```

Comment:
This would move to a `Kconfig` file with content:
```
config FLASH_RAM
    bool
    default y

config FLASH_RAM_UNALIGNED_READ
    bool
    default y
```

Reply:
Do not understand the comment.

Reply:
> Do not understand the comment.

the comment is following this comment: https://github.com/nrfconnect/sdk-nrf/pull/5620#discussion_r709154941

So that if the setting is promptless, and defaulting to `n`, then elsewhere in the Kconfig tree we can enable it conditionally by using:
```
if <SOMETHING>

config FLASH_RAM
    bool
    default y

config FLASH_RAM_UNALIGNED_READ
    bool
    default y

endif
```

that `<SOMETHING>` could be the DFU enabling, or mcuboot on thingy53, or something else.
That way it can be guaranteed that requirements to the flash simulator stub is met when a given configuration is enabled.

Relying on settings that has strong dependencies on each other in prj.conf can be fragile if there are no other checks in place.

Reply:
fixed